Main.java

```diff
@@ -31,11 +31,21 @@ private Character getChar() throws IOException {
             String input = stdinReader.readLine().trim();
 
             if (input.length() == 1) {
+                if (guesses.contains(input.charAt(0))) {
+                    System.out.println("This character was already guessed! Try another one.");
+                    printGuesses();
+                    printBoard();
+//                    ch = '\0';
```

Comment:
??

Reply:
Checks if guess is unique and reprints the board so it looks clean.

Reply:
i mean the comment